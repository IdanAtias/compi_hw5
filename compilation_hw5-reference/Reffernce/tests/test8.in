	while(true)
	{
		print "#25) contribute to our plague.";
		while(not false)
		{
			print "#26) Break all your promises";
			while(2 < 2 or 1 > 2 or 3<4)
			{
				print "#27) tear down this steadfast wall,";
				while(6 >= 6)
				{
					break;
				}
				break;
			}
			print "#28) restraints are useless here,";
			break;
		}
		break;
	}
	print "#29) tasting salvation's near.";
	if(true) { int comTruise; }
	else { int comTruise; }