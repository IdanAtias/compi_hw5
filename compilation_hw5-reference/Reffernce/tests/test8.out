#25) contribute to our plague.
#26) Break all your promises
#27) tear down this steadfast wall,
#28) restraints are useless here,
#29) tasting salvation's near.
